src/components/AnacondaPage.jsx

```diff
@@ -14,12 +14,20 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
-import React, { cloneElement, useState } from "react";
+import React, { cloneElement, useEffect, useState } from "react";
 import { Alert, Stack, Title } from "@patternfly/react-core";
 
-export const AnacondaPage = ({ children, step, title }) => {
+export const AnacondaPage = ({ children, setIsFormDisabled, step, title, usePageInit }) => {
     const [stepNotification, setStepNotification] = useState();
 
+    // If there is usePageInit custom hook for the page, call it
+    usePageInit?.();
+
+    // Otherwise just set the form to enabled so that the user can interact with it
+    useEffect(() => {
+        if (!usePageInit) setIsFormDisabled(false);
+    }, [setIsFormDisabled, usePageInit]);
```

Comment:
The `setIsFormDisabled` dependency here is really weird -- that function never changes. It would make more sense to depend on changes to the actual value. But I suppose that's just  React not understanding the bigger picture, so just a formality -- and the old code you removed had it as well. `</thinking-aloud>`.

Reply:
It's just formality to silence the linter indeed.